UnityProject/Assets/Scripts/Learning/UI/ProtipListUI.cs

```diff
@@ -1,14 +1,14 @@
-using System;
 using TMPro;
 using UnityEngine;
-using UnityEngine.UI;
 
 namespace Learning
 {
 	public class ProtipListUI : MonoBehaviour
 	{
 		[SerializeField] private GameObject entryToSpawn;
 		[SerializeField] private Transform entryList;
+		[SerializeField] private Transform mainPage;
+		[SerializeField] private Transform settingsPage;
```

Comment:
I don't like this - settings consumers shouldn't communicate directly with the settings UI - leave that up to the settings scripts.

Reply:
what do you mean

Reply:
Apologies, I thought this was linking to the escape-menu settings page 🙂